METRICS.md

```diff
@@ -0,0 +1,282 @@
+# Four Key Metrics Calculations
```

Comment:
Add a sentence that states the purpose of this page. Something like

"This page describes the calculation of each metric that the Four Keys dashboard displays. Read this page if you are contributing to the source code for the metrics calculation, or if you want details about the source of each metric."